Assets/AnimatedMaterials.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AnimatedMaterials : MonoBehaviour
+{
+    // serialize material field list
+    [SerializeField] private List<Material> _animatedMaterials = new List<Material>();
+    [SerializeField] private float _noclipSlowdownFactor = 10f;
```

Comment:
Do you want to put this in the noclipOptions scriptable object, so that we can have a global parameter for all the objects, or do you prefer to keep it like this and customizable for each object?

Reply:
if you apply the modification and everything works as before its ok! i think every material can slow down with the same factor

Reply:
Sorry, this is just in the materials holder, it is not worth to make the change